oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb

```diff
@@ -21,7 +21,7 @@
 
   include Pedant::RSpec::CookbookUtil
 
-  context "DELETE /cookbooks/<name>/<version>" do
+  context "DELETE /cookbook_artifacts/<name>/<version>" do
```

Comment:
Minor quibble but below we call "version" a "cookbook_identifier".  Perhaps that is the better name?